src/commands/test/examples/cheats/test_cheats.cairo

```diff
@@ -102,3 +102,38 @@ func test_clearing_mocks{syscall_ptr : felt*, range_check_ptr}(contract_address
     let (res) = ITestContract.get_felt(EXTERNAL_CONTRACT_ADDRESS)
     return ()
 end
+
+# deploy_contract
+@contract_interface
+namespace BasicContract:
+    func increase_balance(amount : felt):
+    end
+
+    func get_balance() -> (res : felt):
+    end
+end
+
+@external
+func test_deploy_contract{syscall_ptr : felt*, range_check_ptr}():
+    alloc_locals
+
+    local contract_a_address : felt
+    %{ ids.contract_a_address = deploy_contract("./src/commands/test/examples/basic.cairo") %}
```

Comment:
I think `deploy_contract` should return dataclass (contract wrapper) with an address property. It would allow us to expand the interface without introducing breaking changes.

Reply:
That's interesting idea, what could such interface include? Because for now wrapper would be only overhead for a user. We can always introduce a new interface by introducing a new function, so im not sure if simplicity is worth the sacrifice at this point.

Reply:
We could allow user calling methods directly from hints, for example.

`Truffle` uses this approach, if I am not mistaken.

Reply:
Adding new function increases the learning curve.